tests/conftest.py

```diff
@@ -84,15 +84,17 @@ def pytest_collection_modifyitems(config, items):
             reason=f'tests for {cloud} is skipped, try setting --{cloud}')
 
     cloud_to_run = _get_cloud_to_run(config)
+    gc = _generic_cloud(config)
```

Comment:
How about using the full name `generic_cloud` for the variable?

Reply:
I was slightly reluctant since there's a function called `generic_cloud`, but can do!

Reply:
Updated!